easybuild/easyconfigs/e/ESMF/ESMF-8.0.0-foss-2019b.eb

```diff
@@ -0,0 +1,31 @@
+name = 'ESMF'
+version = '8.0.0'
```

Comment:
@lexming Is 8.0.1 incompatible somehow?!

Reply:
@boegel 8.0.0 is already being used as a dep for the 2019b generation, I suspect that's why its used here.

Reply:
@boegel it's not incompatible, this version is just what I added at the time (and tested) in this toolchain, which is the same as its `intel/2019b` counterpart.